scraper.py

```diff
@@ -3,40 +3,45 @@
 from time import sleep
 import concurrent.futures
 from bs4 import BeautifulSoup
+from os import path
 
-import helpers.config as config
 from helpers.listings import Listings
+from helpers.page import Page
 import helpers.email_notification as email_notification
-
-
+import helpers.config as config
 
 def main():
-    reviewed_listings = {}
-    with open(config.json_file) as json_file:
-        reviewed_listings = json.load(json_file)
-
-    listings = Listings(reviewed_listings)
-
-    for i in range(1,1000):
-        page = requests.get(config.get_serach_url(i))
-        soup = BeautifulSoup(page.content, 'html.parser')
-
-        ads = soup.find_all('div', attrs={'class':'oglas_container', 'itemtype': 'http://schema.org/ListItem'})
-        ads_len = len(ads)
-        if ads_len == 0:
-            break
-        with concurrent.futures.ThreadPoolExecutor(max_workers=ads_len) as executor:
-            for ad in ads:
-                executor.submit(listings.parse_listing, ad)
-
-    #email_msg = listings.generate_email()
-    #if email_msg != '':
-    #    email_notification.send(email_msg)
-
-    with open(config.json_file, 'w') as fp:
-        json.dump(listings.get_reviewed_listings(), fp)
-
-
+    page_config: Page
```

Comment:
Never seen anything like this before

Reply:
I wanted to define that page_config is of type Page and not any other type. Only in this case VS Code advised me for properties in Page class.

Reply:
interesting, the more you know